lib/instrumentation/index.js

```diff
@@ -70,30 +70,41 @@ function Instrumentation (agent) {
   this._patches = new NamedArray()
 
   for (let mod of MODULES) {
+    if (!Array.isArray(mod)) mod = [mod]
+    const pathName = mod[0]
+
     this.addPatch(mod, (exports, name, version, enabled) => {
       // Lazy require so that we don't have to use `require.resolve` which
       // would fail in combination with Webpack. For more info see:
       // https://github.com/elastic/apm-agent-nodejs/pull/957
-      const patch = require(`./modules/${mod}`)
+      const patch = require(`./modules/${pathName}`)
       return patch(exports, name, version, enabled)
     })
   }
 }
 
-Instrumentation.prototype.addPatch = function (name, handler) {
-  const type = typeof handler
-  if (type !== 'function' && type !== 'string') {
-    this._agent.logger.error('Invalid patch handler type:', type)
-    return
-  }
+Instrumentation.prototype.addPatch = function (modules, handler) {
+  if (!Array.isArray(modules)) modules = [modules]
 
-  this._patches.add(name, handler)
-  this._startHook()
+  modules.forEach(mod => {
+    const type = typeof handler
+    if (type !== 'function' && type !== 'string') {
+      this._agent.logger.error('Invalid patch handler type:', type)
+      return
+    }
+
+    this._patches.add(mod, handler)
+    this._startHook()
+  })
 }
 
-Instrumentation.prototype.removePatch = function (name, handler) {
-  this._patches.delete(name, handler)
-  this._startHook()
+Instrumentation.prototype.removePatch = function (modules, handler) {
+  if (!Array.isArray(modules)) modules = [modules]
+
+  modules.forEach(mod => {
+    this._patches.delete(mod, handler)
+    this._startHook()
```

Comment:
Again, the call to `this._startHook()` doesn't need to be made between each change, you could just put it after the loop. Also, for/let loop too, please. :)